es/pck.tex

```diff
@@ -4,76 +4,75 @@ \chapter{Conocimiento de la pedagogía del contenido}\label{s:pck}
 
 \begin{description}
 
-\item[\gref{g:content-knowledge}{conocimiento del contenido}]
+\item[\gref{g:content-knowledge}{conocimiento del contenido}],
   por ejemplo, como programar;
 
-\item[\gref{g:general-pedagogical-knowledge}{conocimiento general de pedagogía}]
+\item[\gref{g:general-pedagogical-knowledge}{conocimiento general de pedagogía}],
   por ejemplo, comprensión de la psicología del aprendizaje;
   y
 
-\item[\gref{g:pedagogical-content-knowledge}{conocimiento de la pedagogía del contenido}]
-  (PCK por Pedagogical Content Knowledge en inglés),
-  que es el conocimiento específico de cómo enseñar un concepto particular a una audiencia en particular.
+\item[\gref{g:pedagogical-content-knowledge}{conocimiento de la pedagogía del contenido}],
+  que es el conocimiento específico acerca de cómo enseñar un concepto particular a un público en particular.
   En informática,
-  PCK incluye cosas tales como qué ejemplos usar cuando se enseña, como se incluyen parámetros en una función o qué conceptos erróneos son los más comunes sobre etiquetas HTML anidadas.
+  el conocimiento de la pedagogía del contenido incluye qué ejemplos usar cuando se enseña, cómo se incluyen parámetros en una función o qué conceptos erróneos sobre etiquetas HTML anidadas son los más comunes.
 \end{description}
 
 Podemos agregar conocimiento técnico a este conjunto~\cite{Koeh2013},
-pero eso no cambia el punto clave: no es suficiente saber sobre el tema y como enseñar---tienes que saber cómo enseñar ese tema en particular~\cite{Maye2004}.
-Este capítulo resume algunos resultados de investigaciones sobre enseñanza de informática para añadir a tu colección de PCK.
+pero eso no cambia el punto clave: no es suficiente saber sobre el tema y cómo enseñar---tienes que saber cómo enseñar ese tema en particular~\cite{Maye2004}.
+Este capítulo resume algunos resultados de investigaciones sobre enseñanza de informática para añadir a tu colección sobre el conocimiento de la pedagogía del contenido.
 
 Como con toda investigación,
 se requiere cierta precaución al interpretar los resultados:
 
 \begin{description}
 
 \item[Las teorías cambian a medida que se obtienen más datos.]
-  La investigación en educación en informática (CER, por Computing education research en inglés) es una disciplina nueva:\index{computing education research}
-  la Sociedad Americana de Educación en Ingeniería fue fundada en 1893 y el Consejo Nacional de Profesores de Matemática en 1920, pero la Asociación de Profesores de Informática no se creó hasta 2005.
+  La investigación sobre educación en informática es una disciplina nueva:\index{computing education research}
+  la Sociedad Americana de Educación en Ingeniería fue fundada en 1893 y el Consejo Nacional de Docentes de Matemática en 1920, pero la Asociación de Docentes de Informática no se creó hasta 2005.
```

Comment:
Si el nombre oficial de la asociación usa la palabra Profesores en este caso no la cambiaría porque es el nombre de la institución.  Por ejmplo, aqui El Colegio de Técnicos e Ingenieros se llama así, en masculino (y soy parte aunque no soy no ténica ni ingeniera).

Reply:
se llama Teachers

 _National Council of Teachers of Mathematics in 1920, but the Computer Science Teachers Association didn’t exist until 2005_